test/elements/test_admin.py

```diff
@@ -123,6 +123,7 @@ def test_settings(self, m):
                     "auto_assign_org_role": "Viewer",
                 },
             },
+            headers={"Content-Type": "application/json"},
```

Comment:
That the test cases needed to be adjusted by explicitly adding the `Content-Type: application/json` header all over the place: Isn't that telling us that the code actually introduced a backwards incompatibility?

Maybe that adjustment should more properly be addressed on behalf of the central request dispatcher in `grafana-client`, instead of tweaking all the test cases?

Reply:
Actually no, it's not about Niquests but rather the mocking library that did not add a proper header with json content. And Niquests is stricter than Requests, by that I mean that JSON decoding require a proper content type. 
Short story, a tiny bug in the mocking lib.

Reply:
I see, thanks for clarifying. Not wanting to be overly nitpicky, but still let me ask that question: Can we fix it in the mocking lib? Is it a different one than before? Any pointers welcome...

Reply:
Anyway, please ignore my request to understand. You elaborations are well received, and those spots will be different anyway when we will start introducing integration tests anyway. Thanks again!

Reply:
:heavy_check_mark: I've proposed a fix for this using a patch applied in our "compat" module.